charts/trino/values.yaml

```
@@ -3,11 +3,13 @@
 # Declare variables to be passed into your templates.
 
 image:
-  repository: trinodb/trino
-  pullPolicy: IfNotPresent
+  registry: ""   # Trino image registry
+  repository: trinodb/trino    # Trino image repository
   # Overrides the image tag whose default is the chart version.
   # Same value as Chart.yaml#appVersion
-  tag: 432
+  tag: ""   # Trino image tag. Default is appVersion from Chart.yaml
+  digest: ""   # Trino image digest as `sha256:abcd...`. Please note this parameter, if set, will override the tag
```

```suggestion
  digest: ""   # Optional digest value of the image specified as `sha256:abcd...`. A specified value overrides the tag.
```

Question.. does it also override repository?

---

No the repository is not affected.  If `digest` is defined, it will be used instead of `tag` when building the image reference.